code/main/media/mediaModule.brs

```diff
@@ -68,7 +70,7 @@ function _adb_MediaModule(configurationModule as object, edgeRequestQueue as obj
             mediaHit = m._createMediaHit(requestId, eventType, eventData.xdmData, eventData.tsObject)
 
             if eventType = m._CONSTANTS.MEDIA.EVENT_TYPE.SESSION_START
-                m._sessionManager.createSession(clientSessionId, m._configurationModule, sessionConfig, m._edgeRequestQueue)
+                m._sessionManager.createSession(clientSessionId, m._configurationModule, sessionConfig, m._edgeModule)
```

Comment:
Since the `_edgeModule` instance is used only when creating a new session, can we refactor the`_adb_MediaSessionManager` function to accept an `_edgeModule` instance and remove the `_edgeModule` instance in the media module?
```
  _sessionManager: _adb_MediaSessionManager(edgeModule),
```

Reply:
Similarly, we should also pass configuration module to sessionManager constructor. What do you say @yangyansong-adbe? Can I open a separate PR for that refactor?

Reply:
Actually now that I think of I don't see a real value of this refactor, since we need to pass the configModule and EdgeModule to MediaSession and manager has no use of it. What do you say?

Reply:
Yes, let's discuss and refactor it later after adding the consent module.